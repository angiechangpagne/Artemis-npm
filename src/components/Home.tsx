import * as React from 'react';

const Home : React.FC = () => {
  return (
    <div id='home-component'>
      <h1> ARTEMIS </h1>
      <p>GraphQL query has not been detected yet. </p>
      <div className="loarder">Loading...</div>
    </div>
  )
}
 export default Home;