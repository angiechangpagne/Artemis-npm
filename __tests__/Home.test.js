import * as React from 'react';
import { shallow } from 'enzyme';
import Home from '../src/components/Home';
import renderer from 'react-test-renderer';


const Enzyme = require("enzyme");
const Adapter = require("enzyme-adapter-react-16");

Enzyme.configure({adapter: new Adapter() });

describe('<Home />', () => {
   let wrapper;
   it('Should render <Home />', ()=> {
     wrapper = shallow(<Home />)
     expect(wrapper).toBeDefined();
   })


   it('should render correctly with no props',() => {
     const homeComponent = renderer.create(<Home/>).toJSON();
     expect(homeComponent).toMatchSnapshot();
   })
})
