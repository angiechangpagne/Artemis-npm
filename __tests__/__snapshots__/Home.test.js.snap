//jest snapshot

exports[`<Home/> should render correctly with no props 1`] = `
<div 
id="home-component"
>
<h1> 
ARTEMIS
</h1>
<p> 
GraphQL query has not been detected yet.
</p>
<div 
className="loader"
>
Loading...
</div>
`;